app/controllers/proposal_controller.rb

```diff
@@ -14,6 +16,9 @@ def show
   end
 
   def new
+    @event = @program.events.new
+    @event.event_users.new(user: current_user, event_role: 'submitter') if current_user
```

Comment:
I am using `if current_user` cause same `new` action will be used for non-signed in users[1] and we don't have `current_user` there.

[1] we allow proposal submission for non-signed in user.